tests/infrared/16/metrics-collectd-qdr.yaml.template

```diff
@@ -0,0 +1,56 @@
+---
+tripleo_heat_templates:
+    - /usr/share/openstack-tripleo-heat-templates/environments/metrics/collectd-write-qdr.yaml
+
+custom_templates:
+    parameter_defaults:
+        CollectdAmqpInterval: 5
+        CollectdDefaultPollingInterval: 5
+        CollectdDefaultPlugins:
+            - cpu
+            - df
+            - disk
+            - hugepages
+            - intel_rdt
+            - interface
+            - ipmi
+            - load
+            - memory
+            - processes
+            - unixsock
+            - uptime
+        DnsServers: ["10.19.42.41","10.11.5.19","10.5.30.160"]
```

Comment:
@elfiesmelfie this is the line that results in ability to remove --ntp-server (although I plan to swap back to using the command line. Use of DnsServer was for a non- infrared reployment)